src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/admin/staff/JudgeServiceTest.java

```diff
@@ -177,23 +262,22 @@ private AdminData createAdminData(String judgeCode, String judgeName, String tri
 
     private AdminData createAdminDataWithDynamicList(String id, String tribunalOffice, String code, String name,
                                                      String employmentStatus) {
-        var adminData = createAdminData(code, name, tribunalOffice, employmentStatus);
-        adminData.setTribunalOffice(tribunalOffice);
+        AdminData adminData = createAdminData(code, name, tribunalOffice, employmentStatus);
+        adminData.setJudgeSelectList(getDynamicFixedListType(id, name));
+        adminData.getJudgeSelectList().setValue(DynamicValueType.create(id, name));
+        return adminData;
+    }
 
+    private DynamicFixedListType getDynamicFixedListType(String id, String name) {

```

Comment:
You can probably replace this method with a call to the DynamicFixedListType static factory method .of

Reply:
Done